wsmatrix@martin.zurowietz.de/WorkspaceSwitcherPopup.js

```diff
@@ -14,6 +14,7 @@ class WsmatrixPopupList extends WorkspaceSwitcherPopupList {
       this._columns = columns;
       this._scale = scale;
       this._activeWorkspaceIndex = 0;
+      this._itemSpacingFixed = this._itemSpacing;   //workaround to avoid popup jumps issue#14
```

Comment:
Doesn't this mean that `_itemSpacingFixed` will always equal `0` rather than the actual spacing? I'll have a closer look at this later.

Reply:
yeah, I also thought about another workaround which is going to use the new `_itemSpacing` value, but I am not sure if it will cause any troubles.

```
   _init(rows, columns, scale) {
      super._init();
      this._workspaceManager = DisplayWrapper.getWorkspaceManager();
      let oldList = this._list;
      this._list = new WsmatrixPopupList(rows, columns, scale);
      this._container.replace_child(oldList, this._list);
      this._redisplay();
      this.hide();
      
      this.connect('style-changed', () => {
          this._redisplay();
      });
   }
```
I am calling `this._redisplay();` on `style-changed`,  it will receive the new value and render it

Reply:
I don't know why exactly but it works and seems to be a reasonable fix.